indico/MaKaC/webinterface/pages/admins.py

```diff
@@ -148,35 +149,44 @@ def _setActiveSideMenuItem(self):
     def _getPageContent(self, params):
         return "nothing"
 
+
 class WAdmins(wcomponents.WTemplated):
 
-    def getVars( self ):
-        vars = wcomponents.WTemplated.getVars( self )
+    def getVars(self):
+        wvars = wcomponents.WTemplated.getVars(self)
         minfo = info.HelperMaKaCInfo.getMaKaCInfoInstance()
-        vars["title"] = minfo.getTitle()
-        vars["organisation"] = minfo.getOrganisation()
-        vars['supportEmail'] = Config.getInstance().getSupportEmail()
-        vars['publicSupportEmail'] = Config.getInstance().getPublicSupportEmail()
-        vars['noReplyEmail'] = Config.getInstance().getNoReplyEmail()
-        vars["lang"] = minfo.getLang()
-        vars["address"] = ""
+        wvars["title"] = minfo.getTitle()
+        wvars["organisation"] = minfo.getOrganisation()
+        wvars['supportEmail'] = Config.getInstance().getSupportEmail()
+        wvars['publicSupportEmail'] = Config.getInstance().getPublicSupportEmail()
+        wvars['noReplyEmail'] = Config.getInstance().getNoReplyEmail()
+        wvars["lang"] = minfo.getLang()
+        wvars["address"] = ""
         if minfo.getCity() != "":
-            vars["address"] = minfo.getCity()
+            wvars["address"] = minfo.getCity()
         if minfo.getCountry() != "":
-            if vars["address"] != "":
-                vars["address"] = "%s (%s)"%(vars["address"], minfo.getCountry())
+            if wvars["address"] != "":
+                wvars["address"] = "{0} ({1})".format(wvars["address"], minfo.getCountry())
             else:
-                vars["address"] = "%s"%minfo.getCountry()
-        vars["timezone"] = Config.getInstance().getDefaultTimezone()
-        vars["systemIconAdmins"] = Config.getInstance().getSystemIconURL( "admin" )
-        iconDisabled = str(Config.getInstance().getSystemIconURL( "disabledSection" ))
-        iconEnabled = str(Config.getInstance().getSystemIconURL( "enabledSection" ))
+                wvars["address"] = "{0}".format(minfo.getCountry())
```

Comment:
I think we are going for placeholders without numbering unless really necessary. `'{}'.format()` also works ;)